runtime/src/accounts_index.rs

```diff
@@ -229,10 +239,7 @@ impl<'a, T: 'a + Clone> AccountsIndex<T> {
     ) {
         let roots = &self.roots;
 
-        let mut max_root = Self::get_max_root(roots, &list);
-        if let Some(max_clean_root) = max_clean_root {
-            max_root = std::cmp::min(max_root, max_clean_root);
```

Comment:
innocent looking code does hurt us... Sorry for missing this while reviewing #12194 .

Reply:
No worries, I was an idiot because I did the correct thing here (which is almost exactly the same): https://github.com/solana-labs/solana/blob/master/runtime/src/accounts_index.rs#L107-L110 and then forgot to change this 🤦